apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ir/BackendIr.kt

```diff
@@ -58,6 +58,7 @@ internal data class BackendIr(
       val name: String,
       val alias: String?,
       val type: IntrospectionSchema.TypeRef,
+      val normalizedTypeName: String,
```

Comment:
Nit: "normalized" always makes me think of "normalized" cache. Maybe we could find  another name there? I don't have a lot great ideas. Maybe `modelName` ? Where `Model` could refer to the generated class or interface? I think we don't have that too much in the codebase (yet :-P!).

Reply:
how about just call it `typeName`?

Reply:
Ahah naming is hard 🙃 . `typeName` is taken by Kotlin poet already so `modelName` is less " encumbered" but both work for me!